src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,70 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
```

Comment:
is it important here? Pagination doesn't matter about perPage, it only needs to know the number of items. So why don't calculate it in the App?

Reply:
not fixed 
and your `itemsLengthArray` can be prepared at App or helper variable